backend/subletting/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+from rest_framework import routers
+
+
+#from subletting.views import{}
+
+app_name = "subletting"
+
+routers = routers.DefaultRouter()
```

Comment:
I'd also imagine the URLs is throwing an error because you haven't defined `urlpatterns` (see other apps for this)